### ISSUE TITLE
Expect to support configuration export and import features

### ISSUE THREAD

**Issue by LeonYang99999** [tomlerendu/Websocket-King-issues#6]
Hello Tom:
    In the process of using it, I configured a lot of connections and want to export them for other people to use, currently WebSocket King does not support configuration export and import, looking forward to support, thanks a lot.

**Comment by mouyong** [comment]
we need these feature also.

**Comment by flowt-au** [comment]
Yes, please. That would be great. I use 2 dev machines and being able to export / import the request definitions would be super useful.

**Comment by psyirius** [comment]
Check this one out: https://github.com/psyirius/Rockite

Fixed some issues. But Issues and PRs are welcome!
